### PR TITLE
Add fingerprint for 2021 Kia Niro PHEV - variation of '22

### PR DESCRIPTION
**Pull request by dblacknc**

### CODE REVIEW

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -249,7 +249,10 @@ def init_make(self, CP: car.CarParams):
     HyundaiCarInfo("Kia Niro Plug-in Hybrid 2018-19", "All", min_enable_speed=10. * CV.MPH_TO_MS, car_parts=CarParts.common([CarHarness.hyundai_c])),
     HyundaiCarInfo("Kia Niro Plug-in Hybrid 2020", "All", car_parts=CarParts.common([CarHarness.hyundai_d])),
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:250]:
```suggestion
    HyundaiCarInfo("Kia Niro Plug-in Hybrid 2020-21", "All", car_parts=CarParts.common([CarHarness.hyundai_d])),
```

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -249,7 +249,10 @@ def init_make(self, CP: car.CarParams):
...
   ],
-  CAR.KIA_NIRO_PHEV_2022: HyundaiCarInfo("Kia Niro Plug-in Hybrid 2022", "All", car_parts=CarParts.common([CarHarness.hyundai_f])),
+  CAR.KIA_NIRO_PHEV_2022: [
+    HyundaiCarInfo("Kia Niro Plug-in Hybrid 2021", "All", car_parts=CarParts.common([CarHarness.hyundai_d])),
+    HyundaiCarInfo("Kia Niro Plug-in Hybrid 2022", "All", car_parts=CarParts.common([CarHarness.hyundai_f])),
+  ],
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:255]:
```suggestion
  CAR.KIA_NIRO_PHEV_2022: HyundaiCarInfo("Kia Niro Plug-in Hybrid 2022", "All", car_parts=CarParts.common([CarHarness.hyundai_f])),
```

**Comment by dblacknc** [comment]
On sunnypilot's fork I initially forced the 2019 model (manually selected) - that was not a fingerprint as such.   He does not yet have the 2022 merged, or I would have used that.

In lines 174-180 in interface.py there are values common to all Niro EVs and PHEVs, then in 179-180 a conditional to reduce the max steering torque just for the 2018-2020 PHEVs.   The 2021-22 PHEVs can use a higher torque.

In terms of control parameters, my car behaves the same as a 2022.   However it requires the Hyundai D harness on the camera connector, so perhaps this should be treated as a distinct car model.

**Comment by dblacknc** [comment]
I received some advice that this should be a new car.  That and since I didn't have a development environment set up already (doing that now), I'm going to close this PR, start over and will make a new one - for a new car, when ready... shortly!

**Pull request closed without merging**